doc/Language/regexes.pod6

```diff
@@ -512,10 +512,43 @@ string of non-whitespace characters.
 
 =head1 X<Longest Alternation: C<|>|regex,|>
 
-In regexes branches separated by C<|>, the longest match wins, independent of
-the lexical ordering in the regexes.
+For short, in regexes branches separated by C<|>, the longest token match wins,
+independent of the textual ordering in the regexes. However, what C<|> really
+does is further more.
 
-    say ('abc' ~~ / a | .b /).Str;    # OUTPUT: «ab␤»
+C<|> does not decide which breanch to win after finish the whole match,
+but follows L<longest token match, for short, LTM|
```

Comment:
"follows (…)" → "follows the (…)"

Reply:
@Tyil Thank you for your careful review.

Since I rearrange the text and use some new sentences, please help me check if I still ( :P ) express things in improper way when you're free.